pyscriptjs/package.json

```diff
@@ -17,45 +16,41 @@
     "test:watch": "cross-env NODE_OPTIONS=--experimental-vm-modules jest --watch"
   },
   "devDependencies": {
-    "@rollup/plugin-commonjs": "^17.0.0",
-    "@rollup/plugin-legacy": "^2.2.0",
-    "@rollup/plugin-node-resolve": "^11.0.0",
-    "@rollup/plugin-typescript": "^8.4.0",
-    "@tsconfig/svelte": "^1.0.0",
-    "@types/jest": "^28.1.6",
-    "@types/node": "^18.7.11",
-    "@typescript-eslint/eslint-plugin": "^5.36.0",
-    "@typescript-eslint/parser": "^5.36.0",
-    "autoprefixer": "^10.4.7",
-    "cross-env": "^7.0.3",
-    "eslint": "^8.14.0",
-    "jest": "^28.1.3",
-    "jest-environment-jsdom": "^28.1.3",
-    "prettier": "^2.6.2",
+    "@jest/globals": "29.1.2",
+    "@rollup/plugin-commonjs": "22.0.2",
+    "@rollup/plugin-legacy": "2.2.0",
+    "@rollup/plugin-node-resolve": "14.1.0",
+    "@rollup/plugin-typescript": "8.5.0",
+    "@types/jest": "29.1.2",
+    "@types/node": "18.8.3",
+    "@typescript-eslint/eslint-plugin": "5.39.0",
+    "@typescript-eslint/parser": "5.39.0",
+    "cross-env": "7.0.3",
+    "eslint": "8.25.0",
+    "jest": "29.1.2",
+    "jest-environment-jsdom": "29.1.2",
+    "prettier": "2.7.1",
     "pyodide": "0.21.2",
-    "rollup": "^2.71.1",
-    "rollup-plugin-copy": "^3.4.0",
-    "rollup-plugin-css-only": "^3.1.0",
-    "rollup-plugin-livereload": "^2.0.0",
-    "rollup-plugin-serve": "^1.1.0",
-    "rollup-plugin-string": "^3.0.0",
-    "rollup-plugin-terser": "^7.0.0",
-    "svelte": "^3.48.0",
-    "svelte-check": "^1.0.0",
-    "ts-jest": "^28.0.7",
-    "tslib": "^2.4.0",
-    "typescript": "^4.8.2"
+    "rollup": "2.79.1",
+    "rollup-plugin-copy": "3.4.0",
+    "rollup-plugin-css-only": "3.1.0",
+    "rollup-plugin-livereload": "2.0.5",
+    "rollup-plugin-serve": "2.0.1",
+    "rollup-plugin-string": "3.0.0",
+    "rollup-plugin-terser": "7.0.2",
+    "svelte": "3.50.1",
+    "svelte-check": "2.9.1",
+    "ts-jest": "29.0.3",
+    "tslib": "2.4.0",
+    "typescript": "4.8.4"
   },
   "dependencies": {
-    "@codemirror/commands": "^6.1.1",
-    "@codemirror/lang-python": "^6.0.0",
-    "@codemirror/language": "^6.0.0",
-    "@codemirror/state": "^6.0.0",
-    "@codemirror/theme-one-dark": "^6.0.0",
-    "@codemirror/view": "^6.3.0",
-    "codemirror": "^6.0.0",
-    "sirv-cli": "^1.0.0",
-    "svelte-fa": "^2.4.0",
-    "svelte-promisable-stores": "^0.1.3"
```

Comment:
seems like stores aren't used anymore? Did @antocuni eliminate them all?

Reply:
svelte is a compiler and is normally set as a dev dependency.

stores are from `import { writable } from 'svelte/store';`

There is no use of promisable stores in the PyScript project. 
`import { promisable } from 'svelte-promisable-stores';`

Reply:
thanks, resolving...